TweetBook/Contract/V1/Requests/CreateTagRequest.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace TweetBook.Contract.V1.Requests
+{
+    public class CreateTagRequest
+    {
+        public string tagName { get; set; } 
```

Comment:
In C# it is considered good practice to have all the properties and methods names start with a capital letter.